### PR TITLE
make sure that nextflow workdir is not cleaned up when site upps

### PR DESCRIPTION
**Pull request by kjellinjonas**
Template for nextflow config updated to not remove workdir when run is complete

### CODE REVIEW

File: roles/nextflow/templates/nextflow_miarka_site.config.j2 [hunk 1]
```diff
@@ -1,6 +1,10 @@
 
 cleanup = true
 
+{% if site == "upps" %}
+    cleanup = false
```
Comment (matrulda) [inline roles/nextflow/templates/nextflow_miarka_site.config.j2:5]:
```suggestion
cleanup = false
{% else %}
cleanup = true
{% endif %}
```

Even though I think the last occurrence of a parameter is used, I think we should avoid having two occurrences of the `cleanup` parameter. I would suggest to remove line 2 in this template and instead set `cleanup = True` in an else statement. Something like the above.

**Review by matrulda — CHANGES_REQUESTED** [review]
Great work! Please have a look at my comment.

File: roles/nextflow/templates/nextflow_miarka_site.config.j2 (hunk 1 above)
Comment (kjellinjonas) [reply roles/nextflow/templates/nextflow_miarka_site.config.j2:3]: Good suggestion! I have updated the if statement accordingly.

File: roles/nextflow/templates/nextflow_miarka_site.config.j2 [hunk 2]
```diff
@@ -1,5 +1,9 @@
 
-cleanup = true
+{% if site == "upps" %}
+    cleanup = false
+{% else %}
+    cleanup = true
```
Comment (matrulda) [inline roles/nextflow/templates/nextflow_miarka_site.config.j2:5]: This is a nitpick, but I don't think this line should be indented. It probably works like this, but to keep it tidy and easier to reed I think it should be on "root level".

**Review by matrulda — COMMENTED** [review]
Great, I just have one super minor comment. Let me know what you think.

File: roles/nextflow/templates/nextflow_miarka_site.config.j2 (hunk 2 above)
Comment (kjellinjonas) [reply roles/nextflow/templates/nextflow_miarka_site.config.j2]: Agreed, updated accordingly!

**Review by matrulda — APPROVED** [review]
Great!

**Pull request merged**